src/quo/components/profile/link_card/view.cljs

```diff
@@ -0,0 +1,61 @@
+(ns quo.components.profile.link-card.view
+  (:require [clojure.core :as core]
+            [clojure.string :as string]
+            [quo.components.icon :as icons]
+            [quo.components.markdown.text :as text]
+            [quo.components.profile.link-card.properties :as properties]
+            [quo.components.profile.link-card.style :as style]
+            [quo.components.utilities.social.view :as social]
+            [quo.foundations.colors :as colors]
+            [quo.theme]
+            [react-native.core :as rn]
+            [react-native.linear-gradient :as linear-gradient]))
+
+(defn title
+  [link]
+  (case link
+    :link      "Website"
```

Comment:
Do we need to bake these in to the component? Can you align with design and check that this is a fixed list? It seems like a detail we want to pass in instead?
The card should just handle the UI side of things

Reply:
i.e I think the designs have too many variations and really there is just one type of card here and the details can change 👍

Reply:
Done